d2l-scroll-wrapper.js

```diff
@@ -343,6 +343,12 @@ Polymer({
 			value: 'd2l-tier1:chevron-right'
 		},
 
+		overflowBorder: {
+			type: Boolean,
+			value: false,
```

Comment:
You mentioned in the description that most places have the dashed line.  Should this actually default on?  Or, since we want Booleans to default false, change to something like `no-overflow-border`?  It would mean updating more places I guess (removing the old style mixin code from places that want it on, _and_ setting the variable to false everywhere else)

Reply:
I considered it. I'm also considering turning this into a `overflow-style` where it might be one of `none`, `border`, `gradient` or something like that, and maybe having that also control whether the buttons show up or not.

I think I'd like to defer that decision until I've had a change to chat with Jeff about the overall design of this next week. Introducing this attribute now can be a temporary way to get us off the style mixins, and then if we want to change it later or combine it with a few others that should still be easy to do.

Reply:
Another option I just noticed is that we may be able to use the `show-actions` attribute (which is used to show/hide the scroll buttons) and apply the dashed border whenever that's set. With the exception of game-based-learning's usage (which sets show-actions but doesn't set a dashed border) that may work. Looking into the GBL usage to see if it maybe should be using a dashed border...

Reply:
Ok talked to the GBL folks and they're OK with getting the dashed border as well, which would make GBL look like this:

![Screen Shot 2021-03-19 at 12 44 15 PM](https://user-images.githubusercontent.com/5491151/111816123-d68c3080-88b2-11eb-8bee-deeabd49d41d.png)

So I'm going to update this PR to key off `show-actions` instead of `overflow-border` and I'll remove that attribute. That change will also let me remove the left/right mixins right away. I'll need to wait for the rubrics margin thing to be resolved to remove the `--d2l-scroll-wrapper-h-scroll` mixin.

Reply:
Updated the PR.

Reply:
That sounds good, I think the above looks fine!  So if you don't have `show-actions` you'll get no dashed line or arrows, but you can still scroll?  From a quick search it _seems_ like the only places not showing the arrows is the LMS version, and manager dashboard (https://search.d2l.dev/xref/Brightspace/manager-view-fra/src/d2l/d2l-tab-bar.js?r=83fac7ca#201).  So depending on what those look like, a none option may not even be needed - maybe `show-actions` toggles the arrows versus the gradient.  Not the greatest name for it but maybe we could change it after

Reply:
That's correct. And agreed we maybe don't need to transition to a new thing despite the poor name, unless design decides that there needs to be a none|gradient|buttons or something.